admin/kernel-masterdata-service/src/test/java/io/mosip/kernel/masterdata/test/service/SchemaServiceTest.java

```diff
@@ -322,9 +321,17 @@ public void testDeleteIdentitySchemaFailed() throws Exception {
 	
 	@Test(expected = MasterDataServiceException.class)
 	@WithUserDetails("global-admin")
-	public void testDeleteIdentitySchemaFailedUpdate() throws Exception {		
```

Comment:
Test method naming convention is not followed.
methodName_testCase_testResult

Reply:
taken care